source/extensions/common/wasm/wasm.h

```diff
@@ -54,6 +54,7 @@ using WasmCallback3Void = void (*)(void*, uint32_t, uint32_t, uint32_t);
 using WasmCallback4Void = void (*)(void*, uint32_t, uint32_t, uint32_t, uint32_t);
 using WasmCallback5Void = void (*)(void*, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t);
 using WasmCallback0Int = uint32_t (*)(void*);
+using WasmCallback0Int64 = uint64_t (*)(void*);
```

Comment:
For more complex signatures we are moving to the standard c++ name mangling.  See https://github.com/jplevyak/envoy/blob/3f8ae2c418104ea7132f89a04dd1f0db3058fd2a/source/extensions/common/wasm/wasm.h#L68

Reply:
Is WasmCallback_mj the correct mangling for uint64_t (void*)?

Reply:
yes.  I added comments to the stats-meta PR: see https://itanium-cxx-abi.github.io/cxx-abi/abi.html#mangling-builtin, but TLDR: Z = void, j = uint32_t, l = int64_t, m = uint64_t  
where the return type comes first.